client/signup/steps/woocommerce-install/confirm/index.tsx

```diff
@@ -129,7 +139,14 @@ export default function Confirm( props: WooCommerceInstallProps ): ReactElement
 						</p>
 					) }
 
-					<NextButton onClick={ () => goToStep( 'transfer' ) }>{ __( 'Confirm' ) }</NextButton>
+					<NextButton
+						onClick={ () => {
+							dispatch( submitSignupStep( { stepName: 'confirm' }, { confirm: true } ) );
```

Comment:
Want to prevent people landing on transfer without a confirm step happening. (they can bypass with siteslug param)

Reply:
i
```suggestion
							dispatch( submitSignupStep( { stepName: 'confirm' }, { confirmed: true } ) );
```

Reply:
thanks, renamed this further in the end.